functionary/ui/templates/forms/variable_form.html

```diff
@@ -0,0 +1,62 @@
+{% load widget_tweaks %}
+{# djlint: off #}
+{% if add %}
+    <div class="column is-three-quarters add_variable_form" hx-target="this" hx-swap="outerHTML">
+{% else %}
+    <td colspan="3" class="add_variable_form" hx-target="this" hx-swap="outerHTML">
+{% endif %}
+        <form method="post">
+            {% csrf_token %}
+            <input type="hidden" name="parent_id" value="{{ parent_id }}"/>
```

Comment:
Where does this hidden field stuff get used?  It looks like the parent_id is actually read from the kwargs and not the form (which feels like the correct behavior), and then the looping over `form.hidden_fields` seems like it would always be looping over an empty list.

Reply:
That appears to be correct. Removed.